src/constants.py

```diff
@@ -15,6 +17,16 @@
     "ContentDate:End": "content_date_end",
     "Checksum:Algorithm": "checksum_algorithm",
     "Checksum:Value": "checksum_value",
+    "IsSealed": "sealed_state",
+}
+
+CATALOGUE_SEALED_STATE_MAPPER = {
```

Comment:
@udaykumar1995 Is there a better way to handle the mapping?

Reply:
I am not sure, do you see any drawbacks with this approach? @NISH1001

Reply:
Eh no. I don't see any drawbacks. But I was thinking if dictionary-based is better or function-based approach. For now, let's just go with the current one with dictionary...